src/app.ts

```
@@ -139,982 +138,967 @@
+
+    /**
+     * Fedify does not pass the correct context object when running outside of the request context
+     * for example in the context of the Inbox Queue - so we need to wrap handlers with this.
+     */
+    function ensureCorrectContext<B, R>(
+        fn: (ctx: Context<ContextData>, b: B) => Promise<R>,
+    ) {
+        return async (ctx: Context<ContextData>, b: B) => {
+            const host = ctx.host;
+            if (!ctx.data) {
+                (ctx as any).data = {};
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)